contracts/cw1-subkeys/src/contract.rs

```diff
@@ -127,6 +131,14 @@ where
     Ok(res)
 }
 
+pub fn is_expired(env: &Env, expire: Expiration) -> bool {
```

Comment:
No need to implement this, you can do:

`expire.is_expired(&env.block)`

https://github.com/CosmWasm/cosmwasm-plus/blob/main/packages/cw0/src/expiration.rs#L41-L47